README.md

```diff
@@ -62,6 +63,7 @@ This is release 0.1.0. This release supports the following features:
   * Simple linux-based domain with initramfs
   * implements PVnet interface to access other domains via pv network;
   * impelments PVblock interface which allows to mount block devices to the domains.
+* Configuration to use either NVME or USB as a DomD rootfs storage.
```

Comment:
It would be nice to document how to use NVME ad DomD rootfs.

Reply:
Description was provided in "Flash rootfs image to the nvme storage" section. Should I put a link here for it?

Reply:
Ah, it was added earlier... Okay then.